content/docs/next-release/operator.md

```diff
@@ -881,6 +881,25 @@ kubectl delete jaeger simplest
 Deleting the instance will not remove the data from any permanent storage used with this instance. Data from in-memory instances, however, will be lost.
 {{< /info >}}
 
+# Tracing the operator
+
+The Jaeger Operator is able to generate spans for its own operations. By default, the Jaeger Operator ships with an `operator.yaml` that is already configured to send spans to a Jaeger instance named `jaeger`, located in the same namespace as the operator. To use a different instance, change the `operator.yaml` and adjust the flag `reporter.grpc.host-port` to use the appropriate host.
+
+Note that you must manually provision the Jaeger instance. You can do this after the Jaeger Operator has been initialized. The Jaeger Agent will keep the Operator spans in the internal buffer until it makes a connection to the Jaeger instance. The following Jaeger CR can be used to provision a Jaeger instance suitable for non-production purposes:
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+```
+
+Tracing the operator can be disabled by setting the flag `--tracing-enabled` to `false`.
+
+{{< info >}}
+Currently the Operator Lifecycle Manager (OLM) does not offer a way to configure an operator. As a result, if you install the Jaeger Operator via OLM, collecting traces from the Operator is disabled by default.
```

Comment:
Docs are fine - just a question about how it relates to the PR https://github.com/jaegertracing/jaeger-operator/pull/738:
When you say "disabled by default", I don't see any change to the csv to indicate that tracing has been disabled (e.g. `--tracing-enabled=false`) so wouldn't it still be generating the tracing data but just have no agent to handle it?

Reply:
You are correct. We talked about leaving it enabled for a while and disable before a release. I'll send a PR disabling it for now.

Reply:
So will the above text be changed to "Tracing the operator can be enabled by ...."?

Reply:
Just seen the PR which also removes the agent - so to enable will require more docs about adding agent aswell?

Reply:
Yes, I decided to just remove the sidecar as well. I'll update this PR here to include more info about adding the sidecar.